units/cache_historical_task_data_test.go

```diff
@@ -247,22 +249,24 @@ func TestCacheHistoricalTaskDataJob(t *testing.T) {
 		},
 	} {
 		t.Run(test.name, func(t *testing.T) {
+			tctx := testutil.TestSpan(ctx, t)
+
 			require.NoError(t, db.ClearCollections(task.Collection, taskstats.DailyTaskStatsCollection, taskstats.DailyStatsStatusCollection))
 			if test.pre != nil {
-				test.pre(t)
+				test.pre(tctx, t)
 			}
 
 			j := NewCacheHistoricalTaskDataJob("id", "p0")
 			if test.hasErr {
-				j.Run(context.TODO())
```

Comment:
What was this?

Reply:
do you mean what is the purpose of a `context.TODO()`?

Reply:
yeah, idk why they stuck `context.TODO()`s randomly in various places. At least now there will be fewer of them 😄